.github/workflows/test-deploy-wf-service.yml

```
@@ -27,14 +23,16 @@ on:
         default: 'dev'
       sha:
         type: string
-        description: 'Commit SHA ID'
+        description: 'SHA ID'
         required: true
-        default: ''
+        default: 'dev'
```

actionlint

input "sha" of workflow_call event has the default value "dev", but it is also required. if an input is marked as required, its default value will never be used

(events)